src/units/execution.js

```diff
@@ -58,10 +58,12 @@ export class ExecutionUnit extends mix(BaseUnit).with(HashedInputArrayMixin) {
         return this.input.map((i) => new this.constructor.Template(i));
     }
 
-    setApplication(application) {
+    setApplication(application, propagateUpdates = true) {
```

Comment:
I'd suggest using `omitSettingExecutable = false` instead to avoid having to change any prior usage examples